source/fundamentals.txt

```diff
@@ -16,6 +16,7 @@ Fundamentals
    /fundamentals/enterprise-auth
    /fundamentals/bson
    /fundamentals/crud
+   /fundamentals/run-command
```

Comment:
Suggestion: I think that the page entry should be lower in this list (in terms of importance). I would suggest putting it below Transactions